verified-reviews/loaders/storeReview.ts

```diff
@@ -0,0 +1,54 @@
+// deno-lint-ignore-file
+import { AppContext } from "../mod.ts";
+import { Review } from "../../commerce/types.ts";
+import { createClient } from "../utils/client.ts";
+
+export type Props = {
+  /**
+   * @title Number of reviews 
+   */
+  limit: number;
+};
```

Comment:
```suggestion
export type Props = {
  offset?: number;
  /**
   * @title Number of reviews 
   */
  limit?: number;
};
```

We know that api does not have pagination today, but we can do it at loader using offset.